### DISCUSSION TITLE
Opción de poder multipes pendrive y/o imagen

### DISCUSSION THREAD

**Discussion by germanren** [DesobedienteTecnologico/dtails#27] [Ideas]
Hola, te felicito por tú trabajo, la verdad que está muy bueno.

Estaría bueno poder tener el/los pasos para poder generar mas de un pendrive con dtails sin tener que hacer todo el proceso. Osea después que hizo el "desarmado" del archivo img y se agregaron los programas deseados, poder retomar de ahí para copiar a varios pendrive, o tener la opción de generar un archivo img, y desde ese poder copiarlo, ya que ahora obligatoriamente solo lo hace a un pendrive por vez.

Saludos y muchas gracias por el trabajo y tiempo dedicado,
Pablo

**Comment by DesobedienteTecnologico** [comment]
Hola! Gracias por el feedback :)

Es cierto que tengo que trabajar en esa parte para hacerlo mas amigable para quienes quieran meterlo en varios pendrives.
Actualmente eso es posible hacerlo! Hace un tiempo un usuario tenía la misma pregunta en este issue: https://github.com/DesobedienteTecnologico/dtails/issues/18#issuecomment-1613490165

Para llevarlo a cabo:

1º Paso: Modifica esta línea en tu archivo dtails.py https://github.com/DesobedienteTecnologico/dtails/blob/028cd5ae4f3fd45996daca2a65d0e1d94816abbc/dtails.py#L174 . Cambia de **disabled** a **normal**.
2º Paso: Asegúrate de tener el directorio **shared_with_chroot** creado justo después de haber hecho tus modificaciones (Sin haber modificado nada), y empieza a flashear a todos los pendrives que quieras!

Voy a crear un Issue con esto para implementarlo y llevarlo a cabo.

Un placer saber que te gusta y utilizar DTails @germanren .
Saludos
